src/main/java/com/mparticle/kits/ButtonKit.java

```diff
@@ -103,13 +82,53 @@ public void onResult(@Nullable Intent intent, @Nullable Throwable throwable) {
      * <li>Always include this value when reporting orders to your order API.</li>
      * </ul>
      *
+     * <pre>
+     * ButtonKit button = (ButtonKit) MParticle.getInstance()
+     *         .getKitInstance(MParticle.ServiceProviders.BUTTON);
+     * if (button != null) {
+     *     String attributionToken = button.getAttributionToken();
+     *     if (attributionToken != null) {
+     *         // Use token with your order reporting.
+     *     }
+     * }
+     * </pre>
+     *
      * @return the last tracked Button attribution token.
      **/
     @Nullable
     public String getAttributionToken() {
         return merchant.getAttributionToken(applicationContext);
     }
 
+    @Override
+    public void onAttributionTokenChanged(@NonNull String token) {
+        final Map<String, String> attributes = getIntegrationAttributes();
+        attributes.put(ATTRIBUTE_REFERRER, token);
+        setIntegrationAttributes(attributes);
```

Comment:
I get that this in parity with the old code but where in the dashboard do these attributes reside? Is it per user, per application, per session? The javadoc wasn't quite clear on this.

Reply:
No idea